### ISSUE TITLE
Signal Proposal: read and comment at the same time

### ISSUE THREAD

**Issue by traumschule** [Joystream/pioneer#4114]
Taken at https://pioneerapp.xyz/#/proposals/preview/83

As CM i need to be able to comment on the Signal to be approved while writing a comment. So far i need multiple tabs/window of the same proposal or draft in a separate editor.

Remedy: make the sidebar view non-blocking. (This is a Pioneer-specific design decision to control what user can see/do not only here)

![signal-comment](https://user-images.githubusercontent.com/31551045/214329502-a4940207-4867-4d5e-b322-f75b7e72e64c.png)

**Comment by dmtrjsg** [comment]
The right solution would be for the tab to push the content to the left if have space, but this needs to be done together with 
- #1794 

Right now the app would not allow this as you rightly noted..

**Comment by traumschule** [comment]
Yes and it still wouldn't solve the need to comment while reading.
